src/Form/matching-configuration.js

```diff
@@ -252,9 +252,9 @@ const matchingConfiguration = {
         /** @type {DDGMatcherConfiguration} */
         ddgMatcher: {
             matchers: {
-                email: {match: '.mail', forceUnknown: 'search'},
+                email: {match: '.mail', forceUnknown: 'search|filter|subject'},
```

Comment:
This is interesting, perhaps forceUnknown should not require an initial match then - basically making `match` completely optional

Reply:
I tried that, but it made a lot of test fail. I'm looking into it again, because I think I know why the tests were failing (some `forceUnknown` should actually have been `skip`). Looking…

Reply:
Yep, it worked. I've just pushed an update to check `forceUnknown` before anything else and updated the configs to use `forceUnknown` more accurately (sometimes it should have been `skip`). 🎉